catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/endpoints.xml

```diff
@@ -0,0 +1,85 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<!--
+=============================================================
+Endpoints.xml
+Services (HTTP/WS) that catalog-ui-search provides for the UI
+=============================================================
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:jaxrs="http://cxf.apache.org/blueprint/jaxrs">
+
+    <!--
+    =============================================================
+    SIMPLE APPS (AUTH/BUILD/CATALOG/PLATFORM/FEEDBACK/GEOCODER/LOGOUT/SESSION/USER)
+    DESCRIPTORS APPLICATION
+    LIST APPLICATION
+    METACARD APPLICATION
+    QUERY APPLICATION
+    SEARCH FORMS APPLICATION
+    =============================================================
+    -->
+
+    <service ref="sparkServlet" interface="javax.servlet.Servlet">
+        <service-properties>
+            <entry key="urlPatterns" value="/search/catalog/internal/*"/>
+            <entry key="async-supported" value="true"/>
+        </service-properties>
+    </service>
+
+    <service ref="socketServlet" interface="javax.servlet.Servlet">
+        <service-properties>
+            <entry key="urlPatterns" value="/search/catalog/ws"/>
+            <entry key="async-supported" value="true"/>
+        </service-properties>
+    </service>
+
+    <!--
+    =============================================================
+    PROXYING
+    =============================================================
+    -->
+    <!-- TODO Proxy stuff that might be related to the proxy data in the config app, but not sure -->
+
+    <service interface="javax.servlet.Servlet">
+        <service-properties>
+            <entry key="alias" value="/search/catalog/proxy"/>
+            <entry key="servlet-name" value="uiProxyServlet"/>
+            <entry key="async-supported" value="true"/>
+        </service-properties>
+        <ref component-id="servlet"/>
+    </service>
+
+    <!--
+    =============================================================
+    METACARD EDIT ENDPOINT
+    =============================================================
+    -->
+
+    <jaxrs:server id="workspaceService" address="/search/catalog">
+        <jaxrs:serviceBeans>
+            <bean class="org.codice.ddf.catalog.ui.searchui.standard.endpoints.MetacardEditEndpoint">
```

Comment:
_sigh_ we should probably just rip out metacardedit endpoint. it just sits here, gathering unknown dust. You don't have to do it in this pr, but i'm moreso just consoling myself and can rip it out in another.

Reply:
Issue has been made: https://github.com/codice/ddf/issues/4660